doc/nrf/libraries/bin/lwm2m_carrier/requirements.rst

```diff
@@ -35,12 +35,12 @@ Following are some of the requirements and limitations of the application while
 
 * The LwM2M carrier library uses the TLS socket for FOTA.
 
+  * The application can still use one DTLS session and one TLS session (or two DTLS sessions).
   * If the application is using the TLS socket, it must immediately close it when the :c:macro:`LWM2M_CARRIER_EVENT_FOTA_START` event is received.
   * The socket is released by the library again upon the next reboot, or in the event of a FOTA error.
   * If the application always needs a TLS socket, it can use `Mbed TLS`_.
 
-* When in the Verizon network, the LwM2M carrier library uses both the DTLS sessions made available through the modem.
-  Therefore, the application cannot run any DTLS or TLS sessions.
+
```

Comment:
Remove extra empty lines.

Reply:
Thanks, noticed a neighbouring bullet point which could also be removed!